auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -189,17 +180,14 @@ public class AuthenticationAPIClient @VisibleForTesting(otherwise = VisibleForTe
      * Log in a user using a token obtained from a Native Social Identity Provider, such as Facebook, using ['\oauth\token' endpoint](https://auth0.com/docs/api/authentication#token-exchange-for-native-social)
      * The default scope used is 'openid'.
      * Example usage:
-     * <pre>
-     * `client.loginWithNativeSocialToken("{subject token}", "{subject token type}")
-     * .start(new Callback<Credentials, AuthenticationException>() {
-     * {}Override
-     * public void onSuccess(Credentials payload) { }
      *
-     * {}Override
-     * public void onFailure(AuthenticationException error) { }
-     * });
-    ` *
-    </pre> *
+     * ```
+     * client.loginWithNativeSocialToken("{subject token}", "{subject token type}")
+     *     .start(object: Callback<Credentials, AuthenticationException>() {
```

Comment:
I think this doesn't take the `()`
![image](https://user-images.githubusercontent.com/3900123/104053813-377cf580-51ec-11eb-9281-8cb71a84e59f.png)

```suggestion
     *     .start(object: Callback<Credentials, AuthenticationException> {
```